Sources/GRPC/ClientCalls/BidirectionalStreamingCall.swift

```diff
@@ -33,12 +34,19 @@ public final class BidirectionalStreamingCall<RequestMessage: Message, ResponseM
 
   public init(connection: ClientConnection, path: String, callOptions: CallOptions, errorDelegate: ClientErrorDelegate?, handler: @escaping (ResponseMessage) -> Void) {
     self.messageQueue = connection.channel.eventLoop.makeSucceededFuture(())
+    let requestId = UUID()
+    let logger = Logger(subsystem: .clientChannelCall, requestId: requestId)
+    logger.info("making bidirectional streaming request to '\(path)'", metadata: [
```

Comment:
Why is `path` part of the message, but the others are part of the metadata?

Reply:
Actually I think the request and response type should probably be in the message.